Whiskey/Whiskey.psd1

```diff
@@ -144,7 +144,7 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
-* Fixed: PowerShell modules would fail to download if they were found in more than one registered PowerShellGet repository.

```

Comment:
This release note change should stay as a Whiskey release hasn't happened since this change.